Linux Script/configure-linux.sh

```diff
@@ -932,7 +935,7 @@ checkIfTLS()
         if [ "$SUPPRESS_PROMPT" == "false" ]; then
 	        while true;
 			do
-	            read -p "Hey you are going to setup system logs in insecure mode. Do you want to overwrite this with secure mode? (yes/no)" yn
+	            read -p "Hey, you are going to setup the system logs in insecure mode. Do you want to overwrite this with secure mode? (yes/no)" yn
```

Comment:
@Shwetajain148 what conditions cause this message to be displayed?

Reply:
@mostlyjason, This prompt will occur whenever the user will run the script in insecure mode or the script is going to set the NON-TLS configuration i.e. port 514. 

If the user enter **yes** then it means he wants to setup secure logging so the TLS configurations will be copied i.e. port **6514** otherwise the insecure configurations will be copied i.e. port **514** on entering **no**.

Reply:
The problem seems unclear to me can you remove this change and put in a separate JIRA issue?